### ISSUE TITLE
a11y: Use more prominent :focus style

### ISSUE THREAD

**Issue by fhemberger** [Gargron/mastodon#135]
The login screen is very hard to navigate by keyboard only, as the `:focus` styles only give a very minimal hint, there are even elements which lack a `:focus` style completely. Maybe we can come up with something more fitting.

Example: Login screen with `background: rgba(0,0,0,0.25);`:

![bildschirmfoto 2016-11-03 um 15 00 18](https://cloud.githubusercontent.com/assets/153481/19968790/4e820506-a1d6-11e6-91aa-ca35fe549921.png)
